### ISSUE TITLE
Multiselect displaying several clear buttons

### ISSUE THREAD

**Issue by SOAPkelele** [mantinedev/mantine#3692]
### What package has an issue

@mantine/core

### Describe the bug

Multiselect input field displays both crosses when searchable and clearable properties are chosen

<img width="505" alt="image" src="https://user-images.githubusercontent.com/37153107/222978883-8308dfd5-4363-4145-b563-f563ef8a3dae.png">

### What version of @mantine/hooks page do you have in package.json?

6.0.0

### Do you know how to fix the issue

No

### Are you willing to participate in fixing this issue and create a pull request with the fix

No

**Comment by rtivital** [comment]
In which browsers does the second clear button appears?

**Comment by SOAPkelele** [comment]
> In which browsers does the second clear button appears?

google chrome Version 110.0.5481.177 (Official Build) (arm64)

**Comment by 7iomka** [comment]
I also have this unexpected cross icon even in v5
It appears when you try to search in creatable searchable multiselect
<img width="417" alt="image" src="https://user-images.githubusercontent.com/18473137/223062503-d09f9eaa-fc55-4c7b-897f-8811cac53361.png">

**Comment by 7iomka** [comment]
And this is why
<img width="1738" alt="image" src="https://user-images.githubusercontent.com/18473137/223063095-a3b2d916-aaea-498c-a526-35f8f33b5a9f.png">

**Comment by SOAPkelele** [comment]
> And this is why How it looks on user side <img alt="image" width="1738" src="https://user-images.githubusercontent.com/18473137/223063095-a3b2d916-aaea-498c-a526-35f8f33b5a9f.png">
> 
> And this is how it looks on mantine demo
> 
> <img alt="image" width="1661" src="https://user-images.githubusercontent.com/18473137/223063894-4a12a263-9c4b-4a54-932f-0740bab36024.png">

in demo clearable and searchable are not used together

**Comment by jameszhang-a** [comment]
@SOAPkelele Could you share the code for the first screenshot please? I can't reproduce this bug.

**Comment by SOAPkelele** [comment]
@jameszhang-a sure, here it is - https://github.com/SOAPkelele/multiselect-bug/blob/main/src/components/MainBlock.tsx 
I've made a repository which reproduces this error for me, check it out - https://github.com/SOAPkelele/multiselect-bug

**Comment by 7iomka** [comment]
@lgaspari Wow.
Thank you for this research.
I use mantine with tailwind, and I have used tailwind preflight (customized), so to prevent conflicts, I not used `withNormalizeCSS`, and therefore received the same behaviour. 
As a workaround, I just added in `Global` styles this part
```
       /**
         * Reset styles which missed in preflight but required for good look in mantine
         */

        // Hide native elements for search input (mantine has its own setup)
        '[type=search]': {
          appearance: 'none',
        },
        '[type=search]::-webkit-search-cancel-button, [type=search]::-webkit-search-decoration': {
          appearance: 'none',
        },
        // Hide inner focus effect in Firefox
        'button::-moz-focus-inner, [type=button]::-moz-focus-inner, [type=reset]::-moz-focus-inner, [type=submit]::-moz-focus-inner, input[type="file"] > input[type="button"]::-moz-focus-inner':
          {
            borderStyle: 'none',
            padding: 0,
          },
```

I think this is a good solution for many people like me who don't use `withNormalizeCSS`.

**Comment by rtivital** [comment]
This issue is closed for one of these reasons:

- It was fixed in 7.0
- It is no longer reproducible in 7.0
- It is no longer applicable in 7.0
- It does not have a reproduction

If you think that this issue was closed by mistake, and it is still an issue in version 7.0, please reopen it.
